src/program/snabbvmx/doc/README.configuration.md

```diff
@@ -0,0 +1,169 @@
+# Configuration
+
+## SnabbVMX's configuration file
+
+SnabbVMX has its own configuration file (Example: **snabbvmx-lwaftr-xe0.cfg**).
+Configuration file is actually a Lua file, that is processed by `snabbvmx lwaftr`
+command.
+
+It contains a reference to a Snabb's **lwAFTR** configuration file (which
+contains a reference to a binding-table).
+
+**snabbvmx-lwaftr-xe0.cfg**
+
+```lua
+return {
+   lwaftr = "snabbvmx-lwaftr-xe0.conf",
```

Comment:
This is an example of naming config files that I use in the vmx-docker-lwaftr container ([https://github.com/mwiget/vmxlwaftr](url) and indeed chosen to easily match configs together. Anyone is free to chose other naming schema's based on their preference. There are no constraints imposed really.